buildchain/buildchain/iso.py

```diff
@@ -79,6 +79,7 @@ def task_populate_iso() -> types.TaskDict:
             '_iso_render_bootstrap',
```

Comment:
I think the change to this files belong to the other commit (`buildchain: Add task to copy upgrade script`)

Reply:
I found the culprit...It is fixed now

Reply:
Hum, I'm afraid that something went wrong.

The changes are still in this commit, and you did a merge in your PR (which we usually avoid inside a PR).